pkgs/development/python-modules/pyext/default.nix

```diff
@@ -1,15 +1,20 @@
-{ stdenv, buildPythonPackage, fetchPypi }:
+{ stdenv, buildPythonPackage, fetchFromGitHub }:
 
 buildPythonPackage rec {
-    name = pname + "-" + version;
+    name = "${pname}-${version}";
     pname = "pyext";
-    version = "0.6";
+    version = "0.8";
 
-    src = fetchPypi {
-      inherit pname version;
-      sha256 = "6c406cf71b991e1fc5a7f963d3a289525bce5e7ad1c43b697d9f5223185fcaef";
+    src = fetchFromGitHub {
```

Comment:
Is this no longer distributed on `PyPi` or is it behind?

Reply:
Code author stopped uploading to PyPi or tagging releases properly.

Reply:
> Code author stopped uploading to PyPi or tagging releases properly.

Right :+1: 

Can you add a comment with that?